src/scss/dp/overrides/components/_search.scss

```diff
@@ -149,19 +165,53 @@
       height: ($baseline * 5);
       padding-top: ($baseline / 2);
     }
+    //
+    //&--body {
+    //  height: 52px;
+    //  width: 52px;
+    //  margin-top: -2px;
+    //  float: left;
+    //}
+  }
+  &__item {
+    list-style: none;
+    padding-left: 0;
+  }
+}
 
-    &--body {
-      height: 52px;
-      width: 52px;
-      margin-top: -2px;
-      float: left;
-    }
+.search-results {
+
+}
+
+.search-summary {
+  width: 100% !important;
```

Comment:
`!important` should not be used only if you are overwriting the library in this situation I think is not a case